<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>initLog</class>
 <widget class="QWidget" name="initLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>320</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>85</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>85</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>85</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>initLog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>账号类型：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combSelect">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #ffffff;

border-radius:6px;
min-width:80px</string>
   </property>
   <item>
    <property name="text">
     <string>普通用户</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>管理员</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="linePassword">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>140</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0);
border: 1px solid #000000;
background-color: rgb(255, 255, 255);
border-radius:10px;
</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{

	font: 14pt &quot;宋体&quot;;
}
</string>
   </property>
   <property name="text">
    <string>人脸识别签到系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineUser">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>140</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0);
border: 1px solid #000000;
background-color: rgb(255, 255, 255);
border-radius:10px;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLog">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(255, 255, 255);
		border: 2px solid #55aaff;
		border-radius:15px;
		background-color:rgb(58, 170, 255);
		min-width:80px
}
QPushButton:hover{ 
background-color: rgb(0, 52, 150);
border:2px solid #55aaff;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(58, 170, 255);
border:2px solid #55aaff;
border-radius:15px;
}
</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOut">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(0,0,0);
		border: 2px solid #000000;
		border-radius:15px;
		min-width:80px
}QPushButton:hover{ 
color:rgb(255, 255, 255);
background-color: rgb(200, 200, 200);
border:2px solid  #b0b0b0;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border:2px solid  #b0b0b0;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnForget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		border: 2px solid #aaffff;
	
		border-radius:6px;		
		min-width:80px
}
QPushButton:hover{ 
color:rgb(255, 255, 255);
background-color: rgb(200, 200, 200);
border:2px solid  #aaffff;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border:2px solid  #aaffff;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>忘记密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAlter">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		border: 2px solid #aaffff;
	
		border-radius:6px;		
		min-width:80px
}
QPushButton:hover{ 
color:rgb(255, 255, 255);
background-color: rgb(200, 200, 200);
border:2px solid  #aaffff;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border:2px solid  #aaffff;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
